### ISSUE TITLE
``--show-capabilities`` and ``--filter-by-language`` should not necessarily take arguments

### ISSUE THREAD

**Issue by Asnelchristian** [coala-analyzer/coala#2567]

**Comment by arafsheikh** [comment]
`--filter-by-language` is supposed to take arguments to filter stuff. I don't see how it is supposed to work without it.

**Comment by Asnelchristian** [comment]
maybe we should make it in such a way that when no argument is passed it prints all the languages supported and their bears :swear_smile:

**Comment by arafsheikh** [comment]
Yes is `--show-capabilities` is used alone then it should print capabilities of all bears. I think we should introduce a `--filter-by-bear` option too.

**Comment by Makman2** [comment]
but indeed `--filter-by-language` without parameters is weird :3
@Asnelchristian do you agree changing the title to --> ```--show-capabilities`` should not necessarily take arguments` ?

**Comment by Hydro8182** [comment]
Mind if I give this one a shot?
